### ISSUE TITLE
Make a relation User to CPT - one to many or many to many

### ISSUE THREAD

**Issue by its4youb** [CrocoBlock/suggestions#3441]
Hello Crocoblock team, is there something ready or in development to make a relation between Users and CPT?

It functionally will be very welcome, please, could you focus on finding a solution for us?

Example One to Many:

Parent CPT => House

Childreen ==> Users

So many users could be in a same House.

I know that I can create a CPT User and make the relation, but in this case, it is not possible, due to each User need to have the login and password to her/his account.

Pleases, help us.

**Comment by its4youb** [comment]
Need it too +1

**Comment by kochhase** [comment]
Yes and please have a look at the other "user" problems we have for several month.

https://github.com/Crocoblock/suggestions/issues/2110

https://github.com/Crocoblock/suggestions/issues/2131

https://github.com/Crocoblock/suggestions/issues/2222

https://github.com/Crocoblock/suggestions/issues/2226

https://github.com/Crocoblock/suggestions/issues/2390

https://github.com/Crocoblock/suggestions/issues/2410

https://github.com/Crocoblock/suggestions/issues/2241

https://github.com/Crocoblock/suggestions/issues/2415

It´s a big problem, so some of us have to change to pods/ acf pro metabox or toolset.

so please give this a higher priority!

**Comment by milwagh** [comment]
+1 for this 'user' field availability in JetEngine

**Comment by milwagh** [comment]
+1 for this 'user' field availability in JetEngine

**Comment by P258tiwari** [comment]
+1

**Comment by fabio323ti** [comment]
+1 Please add this feature..

**Comment by fabio323ti** [comment]
#4203
